ur_robot_driver/src/hardware_interface.cpp

```diff
@@ -601,6 +755,28 @@ void HardwareInterface::doSwitch(const std::list<hardware_interface::ControllerI
         {
           velocity_controller_running_ = false;
         }
+        if (resource_it.hardware_interface == "hardware_interface::TrajectoryInterface<control_msgs::"
```

Comment:
Yes, that's a strangely verbose type string. Not sure why that's required for these cases. Let's use this right now and simplify this later. Might need some fixes on the [trajectory_interface](https://github.com/fzi-forschungszentrum-informatik/cartesian_ros_control/blob/master/pass_through_controllers/include/pass_through_controllers/trajectory_interface.h#L304).